plugin/src/main/java/org/opensearch/ml/model/MLModelManager.java

```diff
@@ -1364,7 +1391,12 @@ private TokenBucket createTokenBucket(Integer eligibleNodeCount, MLRateLimiter r
                     limit / unit.toSeconds(1),
                     eligibleNodeCount
                 );
-            return new TokenBucket(System::nanoTime, limit / unit.toNanos(1) / eligibleNodeCount, limit, limit / eligibleNodeCount);
+            return new TokenBucket(
+                System::nanoTime,
+                limit / unit.toNanos(1) / eligibleNodeCount,
+                Math.ceil(limit / eligibleNodeCount),
```

Comment:
Why use `ceil`?

Reply:
Because if less than 1 the rate limiter will refuse all request

Reply:
`Changed into max(limit / eligibleNodeCount,1)`